src/voice/voice.controller.spec.ts

```
@@ -22,13 +25,25 @@ const mockStoryService = {
 const mockUploadService = {};
 const mockTextToSpeechService = {
-  batchTextToSpeechCloudUrls: jest.fn(),
+  batchTextToSpeechEager: jest.fn(),
 };
 const mockSpeechToTextService = {};
 const mockVoiceQuotaService = {
   canUseVoice: jest.fn().mockResolvedValue(true),
   getVoiceAccess: jest.fn(),
 };
+const mockStoryQuotaService = {
+  checkStoryAccess: jest.fn().mockResolvedValue({ canAccess: true, reason: 'premium' }),
```

ESLint

[error] Replace `.fn()` with `⏎····.fn()⏎····`

(prettier/prettier)